From bc5da0b99699bbeb653b86398a7112bc0885c31e Mon Sep 17 00:00:00 2001
From: Jim Fehlig <jfehlig@suse.com>
Date: Mon, 6 Jan 2020 18:51:40 -0700
Subject: [PATCH 08/19] libmetrics: Use proper conversion specifier when
 calling log function

From coverity scan

vhostmd-1.1/libmetrics/libmetrics.c:817: invalid_type: Argument "5L" to format specifier "%u" was expected to have type "unsigned int" but has type "long".

Signed-off-by: Jim Fehlig <jfehlig@suse.com>
---
 libmetrics/libmetrics.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libmetrics/libmetrics.c b/libmetrics/libmetrics.c
index 6817df9..0f4cf70 100644
--- a/libmetrics/libmetrics.c
+++ b/libmetrics/libmetrics.c
@@ -823,7 +823,7 @@ static char *get_virtio_metrics(void)
                 usleep(10000);
                 if (time(NULL) > end_time) {
                     libmsg("%s(): Unable to send metrics request"
-                            " - timeout after %us\n", __func__, timeout);
+                            " - timeout after %lis\n", __func__, timeout);
                     goto error;
                 }
             }
@@ -856,7 +856,7 @@ static char *get_virtio_metrics(void)
                 usleep(10000);
                 if (time(NULL) > end_time) {
                     libmsg("%s(): Unable to read metrics"
-                            " - timeout after %us\n", __func__, timeout);
+                            " - timeout after %lis\n", __func__, timeout);
                     goto error;
                 }
             } else
-- 
2.32.0

